### DISCUSSION TITLE
Is there a reliable way to hide panel when visiting base storybook url?

### DISCUSSION THREAD

**Discussion by mayank99** [storybookjs/storybook#14918] [Help]
### The problem

Setting `parameters: { options: { showPanel: false } }` does not work on first story for fresh load (i.e. base url, no cache, etc). Similar to #10109 and #12293 which were closed without a fix/workaround. Also similar to #14442.

What this means is, if you go to the base url (`localhost:6006`), it seems to load with panel open and only after that it redirects to the story url (`localhost:6006?path=/story/firststory--firststory`).

If you go to the story url directly, the panel stays hidden. But that url is too long to share with users easily.

### Workaround?

This issue has plagued us for a while, so were simply closing the panel through some hacky JS code which "clicks" the Hide addons button. But a recent change seems to have made it so that the Shortcuts menu is not part of the DOM tree and cannot be clicked without opening the menu first (which would be bad UX if it happened automatically).

So my question is: **Is there a way to close the addons panel programmatically?** Ideally the `showPanel` parameter would work on fresh load, but I understand that may not be an easy fix.

### Environment info

sb react 6.1.21, chrome 90, windows 10

**Comment by shilman** [comment]
Try setting `showPanel` to `false`: https://storybook.js.org/docs/react/configure/features-and-behavior

**Reply by mayank99** [reply]
That doesn't work on fresh load, which is why I made this post asking for a programmatic workaround that I may be able to call at runtime

**Reply by shilman** [reply]
That's not using parameters, but using `addons.setConfig` in `manager.js`. Did you try it?

**Reply by mayank99** [reply]
I did try it, and it didn't work.

In vacuum, it would be an ok solution, but my use case has a mix of stories where some need the panel and others don't. So even if the panel doesn't open on the very first load (no cache), it breaks for subsequent loads. So if I go to a different story (which opens the panel using parameters), then close storybook, and open the base url again, it loads with panel open (possibly because of some cache).
